lineapy/data/types.py

```diff
@@ -339,6 +343,7 @@ class CallNode(BaseNode):
     # Mapping of global variables that need to be set to call this function
     global_reads: Dict[str, LineaID] = {}
 
+    # TODO: add documentation
```

Comment:
There are some in internals.rst (https://github.com/LineaLabs/lineapy/blob/main/docs/source/references/internals.rst). Want to just link to there for now?

Reply:
Or add a quick summary.

Reply:
this was you 😂 
not sure what the intent was. did you want to add description to implicit_dependencies?

Reply:
if this was in general to document the new way of doing imports (using callnodes instead of importnodes) then linking to internals.rst isnt enough because we dont have that doc updated. We could merge this PR in to add the new behavior before the release and take our time documenting in a separate PR.